### PR TITLE
Updating build process to use buble + buba

### PR DESCRIPTION
**Pull request by davidchase**
I downgraded `eslint` one or two versions because it wouldnt let me add `istanbul` had a peer dep conflict.

related to #101 

thoughts?

### CODE REVIEW

**Comment by briancavalier** [comment]
You may want to try `npm install istanbul@1.1.0-alpha.1` and see if that helps (you have to explicitly install that exact version because it's a prerelease).

**Comment by davidchase** [comment]
ahh ok i will give that a whirl... thanks 👍

**Comment by briancavalier** [comment]
This really looks great, @davidchase.  Are you able to tell if coverage works in this setup?

**Comment by davidchase** [comment]
@briancavalier yup coverage work with your istanbul update
screen shot 2016-07-12 at 9 39 17 pm

**Comment by briancavalier** [comment]
:+1:

**Comment by briancavalier** [comment]
Looks like it needs a rebase, then let's merge it.

File: package.json [hunk 1]
```diff
@@ -28,20 +28,18 @@
   "devDependencies": {
     "@most/eslint-config-most": "^1.0.2",
     "assert": "^1.4.1",
-    "babel-cli": "^6.10.1",
-    "babel-core": "^6.10.4",
     "babel-eslint": "^6.1.2",
-    "babel-plugin-transform-es2015-modules-umd": "^6.8.0",
-    "babel-preset-es2015": "^6.9.0",
+    "buba": "1.0.4",
```
Comment (briancavalier) [inline package.json]: Oops, just noticed this should be a caret version: `"^1.0.5"`

**Comment by davidchase** [comment]
@briancavalier removed scope name from `build-dist`

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
   },
   "scripts": {
     "build": "npm run build-dist && uglifyjs dist/mostPackage.js -o dist/mostPackage.min.js",
-    "build-dist": "mkdir -p dist && rollup -c --name '@most/mostPackage' -o dist/mostPackage.js",
+    "build-dist": "mkdir -p dist && rollup -c --name 'mostPackage' -o dist/mostPackage.js",
```
Comment (briancavalier) [inline package.json:41]: :+1:

**Pull request merged**